docs/modules/ROOT/pages/list-of-metrics.adoc

```diff
@@ -1125,7 +1125,7 @@ the watermark already having passed their windows.
 
 |`cp.map.sizeBytes`
 |count
-|Number of bytes the key-value pairs occupy for this `CPMap`
+|Number of bytes the key-value pairs total for this `CPMap`
```

Comment:
not sure what you are trying to say...is it:

Total number of bytes used by key-value pairs for this `CPMap`

?

Reply:
Yes , sounds better -- done.